src/view.js

```diff
@@ -129,7 +129,7 @@ Marionette.View = Marionette.AbstractView.extend({
   addRegion: function(name, definition) {
     var regions = {};
     regions[name] = definition;
-    return this._buildRegions(regions)[name];
+    return this.addRegions(regions)[name];
```

Comment:
This would create a new object with a 'name' attribute, which is not what's intended, right?